.gitignore

```diff
@@ -0,0 +1,5 @@
+print/attendees.txt
+print/cookies.txt
+print/valid-attendees.txt
+.vagrant
```

Comment:
I'm referring to the .vagrant file specifically. Perhaps I should have said "this".